src/amber/router/cookies/store.cr

```diff
@@ -90,6 +90,9 @@ module Amber::Router::Cookies
             expires : Time? = nil, domain : String? = nil,
             secure : Bool = false, http_only : Bool = false,
             extension : String? = nil)
+      name = URI.encode_www_form(name)
```

Comment:
Is this a breaking change?

Reply:
No this is preserving the existing expected behavior. I had to check the std lib, and 2 years ago the behavior for setting cookie names changed from implicitly encoding names and values to not encoding them at all. So I added this to preserve the expected behavior.

Now that I've said that I need to check and see if this means Amber will only work as expected on Crystal 1.0+

Reply:
Yeah looks like this behavior changed as of 1.0 https://github.com/crystal-lang/crystal/pull/10485